### DISCUSSION TITLE
Qt new versions

### DISCUSSION THREAD

**Discussion by fmarcanoull** [PlusToolkit/PlusLib#1171] [Q&A]
My question is whether the 6.x versions of Qt can be used to compile Plus, as it seems that it is no longer possible to obtain the 5.x.x versions of Qt to configure the compilation environment from scratch. If it's not possible, what alternatives do I have to download and install Qt 5.x.x? Thanks.

**Comment by Sunderlandkyl** [comment]
We will update Plus to use Qt6 once Slicer switches. You can find more information in this post:

https://github.com/PlusToolkit/PlusLib/discussions/1046#discussioncomment-4884593

**Reply by fmarcanoull** [reply]
Thank you Kyle. Is there a way to download version 5.x for windows some site?

**Reply by Sunderlandkyl** [reply]
I haven't tried recently, but it looks like they can still be found here: https://download.qt.io/archive/qt/.

**Reply by fmarcanoull** [reply]
Ok. So this means that no online installation option is available, and the release has to be built instead.

**Reply by Sunderlandkyl** [reply]
I just checked, you can still install Qt 5.X by checking the "Archive" filter button in the online installer.
![image](https://github.com/PlusToolkit/PlusLib/assets/9222709/bd94fb44-6bde-4d4b-b490-a8c0ec6ec965)

**Comment by fmarcanoull** [comment]
Great! The options are a bit hidden, to be honest. Thank you very much!
